packages/components/Button/src/useButton.ts

```diff
@@ -76,7 +76,8 @@ export const useButton = (props: ButtonProps): ButtonInfo => {
   const hasTogglePattern = props.accessibilityActions && !!props.accessibilityActions.find((action) => action.name === 'Toggle');
 
   const theme = useFluentTheme();
-  const shouldUseTwoToneFocusBorder = Platform.OS === ('win32' as any) && props.appearance === 'primary' && !isHighContrast(theme);
+  const shouldUseTwoToneFocusBorder =
```

Comment:
> const shouldUseTwoToneFocusBorder = [](http://example.com/codeflow?start=1&length=36)

At the moment, this is an Officeism. Potentially this will come later to windows but I don't think the windows platform does this today.